{"uid":"e1909e274c0494db","name":"CliKintoneTest-99 Should return the error message when exporting the record with invalid order by query.","fullName":"features/export.feature#CliKintoneTest-99 Should return the error message when exporting the record with invalid order by query.","historyId":"5b8e5072b9c20830eef839f74abd2860:d41d8cd98f00b204e9800998ecf8427e","time":{"start":1705281227634,"stop":1705281231852,"duration":4218},"description":"","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"testStage":{"description":"","status":"passed","steps":[{"name":"Given The app \"app_for_export\" has no records","time":{"start":1705281227634,"stop":1705281228903,"duration":1269},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"And The app \"app_for_export\" has some records as below:","time":{"start":1705281228903,"stop":1705281230160,"duration":1257},"status":"passed","steps":[],"attachments":[{"uid":"be3b65675b16e045","name":"Data table","source":"be3b65675b16e045.csv","type":"text/csv","size":21}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true},{"name":"And Load app ID of the app \"app_for_export\" as env var: \"APP_ID\"","time":{"start":1705281230160,"stop":1705281230160,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"And Load app token of the app \"app_for_export\" with exact permissions \"view\" as env var: \"API_TOKEN\"","time":{"start":1705281230160,"stop":1705281230160,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"When I run the command with args \"record export --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN --order-by 'Unknown_Field desc'\"","time":{"start":1705281230160,"stop":1705281231851,"duration":1691},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"Then I should get the exit code is non-zero","time":{"start":1705281231851,"stop":1705281231851,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"And The output error message should match with the pattern: \"\\[520] \\[GAIA_IQ11] Specified field \\(Unknown_Field\\) not found.\"","time":{"start":1705281231851,"stop":1705281231852,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":7,"attachmentsCount":1,"hasContent":true},"afterStages":[],"labels":[{"name":"host","value":"fv-az573-548"},{"name":"language","value":"javascript"},{"name":"framework","value":"cucumberjs"},{"name":"thread","value":"2362"},{"name":"feature","value":"cli-kintone export command"},{"name":"suite","value":"CliKintoneTest-99 Should return the error message when exporting the record with invalid order by query."},{"name":"tag","value":"@isolated"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["@isolated"]},"source":"e1909e274c0494db.json","parameterValues":[]}